### PR TITLE
Snapshot for cardano-1.28.0

### PR DESCRIPTION
**Pull request by KtorZ**
- [cardano-node 1.28.0](https://github.com/input-output-hk/cardano-node/releases/1.28.0)
- [LTS 18.4](https://www.stackage.org/lts-18.4)

### CODE REVIEW

File: snapshots/cardano-1.28.0.yaml [hunk 1]
```diff
@@ -0,0 +1,175 @@
+name: cardano-1.28.0
+
+resolver: lts-17.6
```
Comment (rvl) [inline snapshots/cardano-1.28.0.yaml:3]:
```suggestion
resolver: lts-17.15
```
Can we use the latest snapshot in the LTS-17 series?

File: snapshots/cardano-1.28.0.yaml [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+packages:
```
Comment (rvl) [inline snapshots/cardano-1.28.0.yaml:5]: I diffed with cardano-1.27.0.yaml and found that some of these packages are older than 1.28.0.

**Review by rvl — COMMENTED** [review]
Cheers @KtorZ

File: snapshots/cardano-1.28.0.yaml (hunk 1 above)
Comment (KtorZ) [reply snapshots/cardano-1.28.0.yaml:3]: _possibly_

File: snapshots/cardano-1.28.0.yaml (hunk 2 above)
Comment (KtorZ) [reply snapshots/cardano-1.28.0.yaml:5]: Ah? It builds locally for me at least. Might not be the latest version of each package available though...

Comment (KtorZ) [reply snapshots/cardano-1.28.0.yaml:5]: Okay, I checked all packages one by one, removed those already in the LTS, and "virtually" separated them in two sections:

- Those not in the LTS
- Those in the LTS but with a different version (either older or more recent)

Few of them are present in LTS-18.3 (latest available), but not in LTS 17.15; so a good third can also be dropped when going for the next major lts.

Comment (rvl) [reply snapshots/cardano-1.28.0.yaml:5]: Oh, if there's LTS-18.x now, then let's use [LTS 18.4](https://www.stackage.org/lts-18.4)[

Also oops I reversed what I meant to say - some of the packages in this snapshot were older than those in `cardano-node-1.27.0.yaml`, but it's fixed now.

**Review by rvl — APPROVED** [review]

**Pull request merged**